src/main/java/seedu/address/model/ReadOnlyResumeBook.java

```diff
@@ -124,6 +129,19 @@
      */
     int getResumeSize();
 
+    /**
+     * Return a Skill item at the specified index from the skill list.
+     * @param index index of skill
+     * @return Skill item at {@code index}
+     */
+    NoteEntry getNoteEntry(Index index);
```

Comment:
try to be consistent with naming and use getNoteEntryByIndex or ById instead, to avoid misuse later on :D